VaticanBAV/et/BAVet196.xml

```diff
@@ -0,0 +1,308 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BAVet196" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Sane Golgotā, Tǝmhǝrta Ḫǝbuʿāt, Malkǝʾa Māryām</title>
+                <editor role="generalEditor" key="AB" />
+                <editor key="AD" />
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV" />
+                        <collection>Aethiopici</collection>
+                        <idno>Aeth. 196</idno>
+                    </msIdentifier>
+
+                    <msContents><summary/></msContents>
+
+                    <physDesc>
+                      <objectDesc form="Codex">
+                        <supportDesc>
+                          <support>
+                            <material key="parchment" />
+                          </support>
+                          <extent>
+                            <measure unit="leaf">80</measure>
+                          </extent>
+                        </supportDesc>
+                      </objectDesc>
+
+                      <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
```

Comment:
I assume that the catalogue does not state why the manuscript is composed of two units, but written by one hand?

Reply:
Actually, there is no notes at all regarding how many hands there are, only about the rubrication. Is there a better way to include information about the rubrication?

Reply:
if this is a true composite MS with 2 distinct parts it is more probable that there are also 2 hands, even if the cataloguers don't specifically mention it... if we take it for granted that we have two units I would have the handDesc with the parts (one each), and possibly add a note with the second handDesc that we do not know if h2 is the same as h1